src/go/k8s/controllers/redpanda/redpanda_controller.go

```diff
@@ -84,6 +84,25 @@ type RedpandaReconciler struct {
 // +kubebuilder:rbac:groups=cert-manager.io,namespace=default,resources=issuers,verbs=get;create;update;patch;delete
 // +kubebuilder:rbac:groups="monitoring.coreos.com",namespace=default,resources=servicemonitors,verbs=get;list;watch;create;update;patch;delete
 
+// additional k8s resources required by flux (Cluster Scope)
+// +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterrolebindings,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterroles,verbs=get;list;watch;create;update;patch;delete
+
+// additional K8s resources (Cluster Scope) defined in rbac.yaml for the redpanda chart
```

Comment:
Can you make this description something auditable, please? Why does it need these at a cluster scope for a Redpanda resource?

Reply:
sure thing